### PR TITLE
Update DG's user stories

### LINKED ISSUE #46: Update DG's user stories

### PR DESCRIPTION
**Pull request by LemonDrew**
Closes #46

### CODE REVIEW

**Comment by kushaldas27** [comment]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -291,18 +291,32 @@ mouse/GUI driven app.
...
+| Novel     | CEO            | receive notifications when candidates complete a new interview stage                             | stay updated on the hiring progress                                                   |
+| Essential | Hiring Manager | view a list of candidates                                                                        | quickly assess who has applied for the position                                       |
+| Essential | Hiring Manager | mark candidates as "hired" or "rejected,"                                                        | finalize the hiring process                                                           |
+| Typical   | Hiring Manager | filter candidates according to different categories e.g. more/less experienced                   | easily pick out the potential candidates.                                             |
+| Essential | Hiring Manager | see the current status of a candidate (e.g., shortlisted, rejected)                              | track their progress                                                                  |
```
Comment (bigismols) [inline docs/DeveloperGuide.md:302]: Comma after e.g can be removed

**Review by oolongwater — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -291,18 +291,32 @@ mouse/GUI driven app.
...
+
+| Priority  | As a …​        | I want to …​                                                                                     | So that I can…​                                                                       |
+|-----------|----------------|--------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| Novel     | CEO            | receive notifications when candidates complete a new interview stage                             | stay updated on the hiring progress                                                   |
+| Essential | Hiring Manager | view a list of candidates                                                                        | quickly assess who has applied for the position                                       |
+| Essential | Hiring Manager | mark candidates as "hired" or "rejected,"                                                        | finalize the hiring process                                                           |
```
Comment (bigismols) [inline docs/DeveloperGuide.md:300]: Should comma after rejected be removed?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -291,18 +291,32 @@ mouse/GUI driven app.
...
+| Typical   | Hiring Manager | see candidates’ salary expectations                                                              | prepare for salary negotiation with the candidate                                     |
+| Typical   | Hiring Manager | search for candidates based on keywords (skills, position, etc.)                                 | quickly find the right profiles                                                       |
+| Novel     | Interviewer    | access a repository of standardized coding questions                                             | structure interviews efficiently and test candidates sufficiently                     |
+| Novel     | Interviewer    | share feedback with other developers                                                             | collaborate and make informed hiring decisions                                        |
+| Typical   | Interviewer    | see candidates’s programming languages                                                           | ensure that they have sufficient programming knowledge based on company needs         |
```
Comment (bigismols) [inline docs/DeveloperGuide.md:312]: candidates's should be candidates'

**Review by bigismols — COMMENTED** [review]
LGTM! other than some minor grammar issues

**Review by oolongwater — APPROVED** [review]
lgtm

**Review by bigismols — COMMENTED** [review]
LGTM!

**Review by SQ77 — APPROVED** [review]
LGTM!

**Pull request merged**